client/src/components/molecules/ConflictDatesAlert/index.tsx

```diff
@@ -14,8 +14,11 @@ const ConflictDatesAlert: FC<ConflictDatesAlertProps> = ({ conflictDates, messag
   const MAX_COLS = 5
   const MAX_DISPLAY = MAX_ROWS * MAX_COLS
 
-  const displayedDates = conflictDates.slice(0, MAX_DISPLAY)
-  const remainingCount = conflictDates.length - displayedDates.length
+  const sortedDates = [...conflictDates].sort(
+    (a, b) => new Date(a).getTime() - new Date(b).getTime()
+  )
```

Comment:
```suggestion
  const sortedDates = [...conflictDates].sort(
    (a, b) => Date.parse(a) - Date.parse(b)
  )
```
new Date() builds a whole object every time, which adds overhead. Date.parse() skips that and gives you the number directly, making it faster and more efficient 🏃